<?xml version="1.0" ?><root>
<report><version>2</version><sessionid>8b7799fbaedfb4d765365ea5</sessionid><type>desync</type><sku>ea.maxis.sims4_64.15.pc</sku><createtime>2023-10-23 22:12:15</createtime><buildsignature>Local.Unknown.Unknown.1.99.264.1030-1.200.000.600.Release</buildsignature><categoryid>rabbit_hole_interactions.py:34</categoryid><desyncid>8b7799fbaedfb4d765365ea5</desyncid><systemconfig/><screenshot/><desyncdata>Exception in Sim Timeline: Exception running Element (AttributeError: 'NoneType' object has no attribute 'get_name')&#13;&#10;Traceback (most recent call last):&#13;&#10;  File "T:\InGame\Gameplay\Scripts\Server\scheduling.py", line 216, in simulate&#13;&#10;  File "T:\InGame\Gameplay\Scripts\Server\alarms.py", line 260, in _run&#13;&#10;  File "T:\InGame\Gameplay\Scripts\Server\scheduler.py", line 321, in _alarm_callback&#13;&#10;  File "T:\InGame\Gameplay\Scripts\Server\careers\career_base.py", line 2328, in _start_work_callback&#13;&#10;  File "T:\InGame\Gameplay\Scripts\Server\careers\career_location.py", line 90, in on_npc_start_work&#13;&#10;  File "T:\InGame\Gameplay\Scripts\Server\sims\university\university_career_tuning.py", line 77, in put_sim_in_career_rabbit_hole&#13;&#10;  File "T:\InGame\Gameplay\Scripts\Server\careers\career_base.py", line 3116, in put_sim_in_career_rabbit_hole&#13;&#10;  File "T:\InGame\Gameplay\Scripts\Server\services\rabbit_hole_service.py", line 223, in put_sim_in_managed_rabbithole&#13;&#10;  File "T:\InGame\Gameplay\Scripts\Server\services\rabbit_hole_service.py", line 161, in _try_put_sim_in_rabbithole&#13;&#10;  File "T:\InGame\Gameplay\Scripts\Server\services\rabbit_hole_service.py", line 633, in _setup_instantiated_travel&#13;&#10;  File "T:\InGame\Gameplay\Scripts\Server\services\rabbit_hole_service.py", line 806, in _push_affordance_with_cancel_callback&#13;&#10;  File "T:\InGame\Gameplay\Scripts\Server\sims\sim.py", line 4066, in push_super_affordance&#13;&#10;  File "T:\InGame\Gameplay\Scripts\Server\interactions\aop.py", line 236, in test_and_execute&#13;&#10;  File "T:\InGame\Gameplay\Scripts\Server\interactions\aop.py", line 224, in execute&#13;&#10;  File "T:\InGame\Gameplay\Scripts\Server\interactions\aop.py", line 212, in execute_interaction&#13;&#10;  File "T:\InGame\Gameplay\Scripts\Server\interactions\interaction_queue.py", line 1333, in append&#13;&#10;  File "T:\InGame\Gameplay\Scripts\Server\interactions\base\super_interaction.py", line 4415, in on_added_to_queue&#13;&#10;  File "T:\InGame\Gameplay\Scripts\Server\interactions\base\interaction.py", line 7454, in on_added_to_queue&#13;&#10;  File "T:\InGame\Gameplay\Scripts\Server\sims\ui_manager.py", line 175, in add_queued_interaction&#13;&#10;  File "T:\InGame\Gameplay\Scripts\Server\sims\ui_manager.py", line 641, in _add_interaction&#13;&#10;  File "T:\InGame\Gameplay\Scripts\Server\rabbit_hole\rabbit_hole_interactions.py", line 34, in get_name&#13;&#10;  File "T:\InGame\Gameplay\Scripts\Server\services\rabbit_hole_service.py", line 512, in get_head_rabbit_hole_home_interaction_nameAttributeError: 'NoneType' object has no attribute 'get_name'rtim=0&#13;&#10;Client Session Time: 60.13&#13; &#10;Num Save Errors: 0&#13; &#10;Num Load Errors: 0&#13; &#10;Current Game State: 0xdd4141d5&#13; &#10;LoadingScreenActiveFlag: 1&#13; &#10;Origin Version: hòB&#13; &#10;Modded: False&#13; &#10;SystemInfo: Windows 10 10.0.19045 GPU: NVIDIA GeForce GTX 1060 3GB nvldumdx.dll 31.0.15.3758, GUID: D7B71E3E-5F42-11CF-5D7D-B3A50EC2D235&#13; &#10;</desyncdata></report>
<report><version>2</version><sessionid>8b7799fbaedfb4d765365ea5</sessionid><type>desync</type><sku>ea.maxis.sims4_64.15.pc</sku><createtime>2023-10-23 23:04:56</createtime><buildsignature>Local.Unknown.Unknown.1.99.264.1030-1.200.000.600.Release</buildsignature><categoryid>carry_elements.py:76</categoryid><desyncid>8b7799fbaedfb4d765365ea5</desyncid><systemconfig/><screenshot/><desyncdata>Exception in Sim Timeline: Exception running Element (RuntimeError: Failed to create new_posture_spec in enter_carry_while_holding!&#13;&#10;&#13;&#10;SI State:&#13;&#10;     running generic_Baththub_ToddlerInfantBath:111377 on object_whirlpool_bathtubJets:0x0c6811b4ea99f7aa[6]&#13;&#10;     running bathtub_InfantBath_Give:111330 on )&#13;&#10;Traceback (most recent call last):&#13;&#10;  File "T:\InGame\Gameplay\Scripts\Server\scheduling.py", line 216, in simulate&#13;&#10;  File "T:\InGame\Gameplay\Scripts\Server\elements.py", line 376, in _run&#13;&#10;  File "T:\InGame\Gameplay\Scripts\Server\carry\carry_elements.py", line 490, in set_up_transition_gen&#13;&#10;  File "T:\InGame\Gameplay\Scripts\Server\carry\carry_elements.py", line 76, in _create_enter_carry_posture&#13;&#10;RuntimeError: Failed to create new_posture_spec in enter_carry_while_holding!&#13;&#10;&#13;&#10;SI State:&#13;&#10;     running generic_Baththub_ToddlerInfantBath:111377 on object_whirlpool_bathtubJets:0x0c6811b4ea99f7aa[6]     running bathtub_InfantBath_Give:111330 on rtim=0&#13;&#10;Client Session Time: 60.13&#13; &#10;Num Save Errors: 0&#13; &#10;Num Load Errors: 0&#13; &#10;Current Game State: 0xdd4141d5&#13; &#10;LoadingScreenActiveFlag: 1&#13; &#10;Origin Version: hòB&#13; &#10;Modded: False&#13; &#10;SystemInfo: Windows 10 10.0.19045 GPU: NVIDIA GeForce GTX 1060 3GB nvldumdx.dll 31.0.15.3758, GUID: D7B71E3E-5F42-11CF-5D7D-B3A50EC2D235&#13; &#10;</desyncdata></report>
</root>